### PR TITLE
Fix incorrect link references

### PR DESCRIPTION
**Pull request by e0i**
Fixes part of the build warnings in #2078.

cc @JohanFreelancer9 for copy review
cc @get-itips @dariomws @tseward 
cc @traya1

### CODE REVIEW

**Review by get-itips — APPROVED** [review]

**Review by JohanFreelancer9 — APPROVED** [review]

File: docset/winserver2012-ps/netsecurity/Rename-NetFirewallRule.md [hunk 1]
```diff
@@ -989,5 +989,5 @@ The path after the pound sign (`#`) provides the namespace and class name for th
 
 [Show-NetFirewallRule](./Show-NetFirewallRule.md)
 
-[New-GPO](../GPv2_Cmdlets/New-GPO.md)
+[New-GPO](../winserver2012-ps/grouppolicy/New-GPO.md)
```
Comment (dariomws) [inline docset/winserver2012-ps/netsecurity/Rename-NetFirewallRule.md:992]:
```suggestion
[New-GPO](../grouppolicy/New-GPO.md)
```

File: docset/winserver2012-ps/iscsi/Get-IscsiConnection.md [hunk 2]
```diff
@@ -352,11 +352,12 @@ The path after the pound sign (`#`) provides the namespace and class name for th
...
 
-[Get-Disk](../Storage2_Cmdlets/Get-Disk.md)
+[Get-Disk](../storage/Get-Disk.md)
 
 [Get-InitiatorPort](../Storage2_Cmdlets/Get-InitiatorPort.md)
 
 [N:MPIO](00000000-0000-0000-0000-000000000000)
 
 [N:Storage](00000000-0000-0000-0000-000000000000)
+[Get-InitiatorPort](../storage/Get-InitiatorPort.md)
```
Comment (dariomws) [inline docset/winserver2012-ps/iscsi/Get-IscsiConnection.md:362]:
```suggestion
[Get-InitiatorPort](../storage/Get-InitiatorPort.md)
```

File: docset/winserver2012-ps/iscsi/Get-IscsiSession.md [hunk 3]
```diff
@@ -353,11 +353,12 @@ The path after the pound sign (`#`) provides the namespace and class name for th
...
 
-[Get-Disk](../Storage2_Cmdlets/Get-Disk.md)
+[Get-Disk](../storage/Get-Disk.md)
 
 [Get-InitiatorPort](../Storage2_Cmdlets/Get-InitiatorPort.md)
 
 [N:MPIO](00000000-0000-0000-0000-000000000000)
 
 [N:Storage](00000000-0000-0000-0000-000000000000)
+[Get-InitiatorPort](../storage/Get-InitiatorPort.md)
```
Comment (dariomws) [inline docset/winserver2012-ps/iscsi/Get-IscsiSession.md:363]:
```suggestion
[Get-InitiatorPort](../storage/Get-InitiatorPort.md)
```

File: docset/winserver2012-ps/iscsi/Get-IscsiTarget.md [hunk 4]
```diff
@@ -228,4 +228,5 @@ The path after the pound sign (`#`) provides the namespace and class name for th
 [N:MPIO](00000000-0000-0000-0000-000000000000)
 
 [N:Storage](00000000-0000-0000-0000-000000000000)
+[Get-InitiatorPort](../storage/Get-InitiatorPort.md)
```
Comment (dariomws) [inline docset/winserver2012-ps/iscsi/Get-IscsiTarget.md:231]:
```suggestion
[Get-InitiatorPort](../storage/Get-InitiatorPort.md)
```

File: docset/winserver2012-ps/iscsi/Update-IscsiTarget.md [hunk 5]
```diff
@@ -263,9 +263,9 @@ The path after the pound sign (`#`) provides the namespace and class name for th
 
 [Get-IscsiTargetPortal](./Get-IscsiTargetPortal.md)
 
-[Get-InitiatorPort](../Storage2_Cmdlets/Get-InitiatorPort.md)
 
 [N:MPIO](00000000-0000-0000-0000-000000000000)
 
 [N:Storage](00000000-0000-0000-0000-000000000000)
+[Get-InitiatorPort](../storage/Get-InitiatorPort.md)
```
Comment (dariomws) [inline docset/winserver2012-ps/iscsi/Update-IscsiTarget.md:270]:
```suggestion
[Get-InitiatorPort](../storage/Get-InitiatorPort.md)
```

**Review by dariomws — CHANGES_REQUESTED** [review]
I'm not sure about Install-WindowsFeature and Uninstall-WindowsFeature... 

I think someone moved these cmdlets to the Server Manager Migration folder because he didn't know where to put them.
The problem with them is that they are available in 2016/2019, not in Windows 10. But I think that the best option here is to have these cmdlets under ServerManager folder in the same way that they are in 2012 and 2012 R2, and we can add a note about that they are not available on Windows 10. Also, I think that Microsoft.Windows.ServerManager.Migration folder should be renamed to MigrationTools. 

These tasks are complimentary, I think. For this PR we can leave these links as @e0i is proposing or point them to the 2012 R2 version.

What do you think @get-itips @tseward?

**Review by dariomws — APPROVED** [review]

**Pull request merged**